run.sh

```diff
@@ -9,8 +9,17 @@ if [ -d models ]; then
 fi
 rm -r eval_*
 
+savePrefix='./models/natural'
+epsStart=1
+rlmodel='DQN'
+restore='None'
+
 if [ -z ${CUDA_VISIBLE_DEVICES} ]; then
-    export CUDA_VISIBLE_DEVICES="0"
+    export CUDA_VISIBLE_DEVICES="0,1,2,3"
```

Comment:
Exporting too many devices is not necessarily helpful (in general, one should enable parallel training explicitly).